utils/answer-types.js

```diff
@@ -1216,6 +1216,7 @@ Khan.answerTypes = $.extend(Khan.answerTypes, {
                         // otherwise correct or not before forwarding on the
                         // message.
```

Comment:
The comment here should probably be changed to reflect the new semantics of what is going on.